composables/usePondProduct.ts

```diff
@@ -0,0 +1,17 @@
+export function usePondProduct() {
+    const { apiClient } = useShopwareContext();
+    const { handleError } = usePondHandleError();
+
+    const getProductsById = async (itemIds: string[]) => usePondCacheAsyncData(`products-${itemIds}`, async () => {
```

Comment:
```suggestion
    const getProductsById = async (itemIds: string[]) => usePondCacheAsyncData(`products-${itemIds.join('-')}`, async () => {
```

right? otherwise its an array?

Reply:
Ups, yes. Done